packages/firebase_performance/android/src/main/java/dev/flutter/plugins/firebaseperformance/FirebasePerformancePlugin.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package dev.flutter.plugins.firebaseperformance;
+
+import io.flutter.embedding.engine.plugins.FlutterPlugin;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.PluginRegistry.Registrar;
+
+/** FirebasePerformancePlugin */
+public class FirebasePerformancePlugin implements FlutterPlugin {
+  private static final String CHANNEL_NAME = "plugins.flutter.io/firebase_performance";
+
+  MethodChannel channel;
+
+  public static void registerWith(Registrar registrar) {
+    final MethodChannel channel = new MethodChannel(registrar.messenger(), CHANNEL_NAME);
+    channel.setMethodCallHandler(new io.flutter.plugins.firebaseperformance.FirebasePerformancePlugin());
```

Comment:
Any reason to not import the package for this class?

Reply:
This was just a placeholder until we decided how we wanted to handle the new embedding packages. I switched to just use the `MethodCallHandler`.